h/services/list_organizations.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h import models
+
+
+class ListOrganizationsService(object):
+
+    """
+    A service for providing a list of organizations.
+    """
+
+    def __init__(self, session):
+        """
+        Create a new list_organizations service.
+
+        :param _session: the SQLAlchemy session object
+        """
+        self._session = session
+
+    def organizations(self, authority=None):
+        """
+        Return a list of organizations filtered on authority.
+        If authority is None, return a list of all organizations.
+        """
+        if authority:
+            return (self._session.query(models.Organization)
+                        .filter_by(authority=authority)
+                        .all())
```

Comment:
The organizations here are still unsorted. Was that intentional in this PR?

Reply:
Ah! Shoot I totally forgot about that! Thanks for catching that!